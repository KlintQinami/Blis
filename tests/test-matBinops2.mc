int main() {
    ivec2 x1 = ivec2(1337, 42);
    ivec2 x2 = ivec2(2, 2);
    ivec2 x3 = x2 * x1 - x1;
    ivec2 x4 = x1 / x2;
    printi(x1.x);
    printi(x1.y);
    printi(x3.x);
    printi(x3.y);
    printi(x4.x);
    printi(x4.y);
    return 0;
}
