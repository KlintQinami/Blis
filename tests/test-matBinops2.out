1337
42
1337
42
668
21
